app.py

```diff
@@ -32,11 +27,19 @@ def sign_up_page():
 
 @app.route('/sign_up', methods=['POST'])
 def sign_up():
-    if signup_utils.process_sign_up():
-        flash('flash: user created successfully.', "info")
+    result = signup_utils.process_sign_up()
+    if result == SignupResults.MISMATCHED_PASSWORD:
```

Comment:
I think we're missing a result? what happens if I try to signup with an existing username?